crates/proof-of-sql/Cargo.toml

```diff
@@ -132,11 +132,7 @@ name = "sushi"
 required-features = [ "arrow" ]
 
 [[example]]
-name = "countries"
-required-features = [ "arrow" ]
-
-[[example]]
-name = "rockets"
```

Comment:
Add these lines 
```rust
name = "countries"
required-features = [ "arrow" ]

[[example]]
name = "rockets"
```
back in and keep 
```rust
[[example]]
name = "tech_gadget_prices"
```

Then the test
```bash
cargo test -p proof-of-sql --no-run --no-default-features --features="test"
```

Should pass and this should be ok, thanks a lot